### PR TITLE
Add :include_blank option to Field::Select

### PR DESCRIPTION
**Pull request by efatsi**
- copy/pasted pretty directly from Field::BelongsTo
- defaults to true similar to BelongsTo configuration

### CODE REVIEW

**Comment by efatsi** [comment]
Had a thought: perhaps a better move would be to set the default to `false` for the new `Fields::Select#include_blank` option. That would mean this PR is not a breaking change, and users could opt-in to including a blank option.

Anyone on the maintenance team have an opinion here?

**Comment by pablobm** [comment]
Thank you for this @efatsi. I agree that we want to avoid unexpected changes of behaviour, and therefore the default should be the same as it is now. Would you be able to make this change?

**Comment by efatsi** [comment]
Can do!

**Comment by efatsi** [comment]
@pablobm just pushed an update. Looks like ruby-25 failed on Circle, although the failure seems unrelated to the changes I've made.

File: spec/administrate/views/fields/select/_edit_spec.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@
       attribute: :email_subscriber,
       data: false,
       selectable_options: [true, false, nil],
+      include_blank_option: false,
```
Comment (pablobm) [inline spec/administrate/views/fields/select/_edit_spec.rb:12]: Wrong option name?

```suggestion
      include_blank: false,
```

Comment (efatsi) [reply spec/administrate/views/fields/select/_edit_spec.rb]: 🤦 yeah, thanks

Comment (efatsi) [reply spec/administrate/views/fields/select/_edit_spec.rb:12]: just kidding, this is correct. The option itself is named `include_blank`, but the getter on the field is `include_blank_option`. Similar to BelongTo field: https://github.com/thoughtbot/administrate/blob/main/lib/administrate/field/belongs_to.rb#L30-L32

Comment (pablobm) [reply spec/administrate/views/fields/select/_edit_spec.rb:12]: Ah, you are right. This is a double and not the real thing, therefore this is a literal method name to stub, as opposed to a constructor argument. Sorry!

**Review by pablobm — APPROVED** [review]
I think I have figured out the issue with CI (see https://github.com/thoughtbot/administrate/pull/2023). Let me double-check it works, merge the fix, and I'll merge this.

**Pull request merged**